### ISSUE TITLE
ggplot with variables from different dataframes

### ISSUE THREAD

**Issue by MayMAAhmed** [STAT545-UBC/Discussion#360]
I was wondering if it's possible to plot 2 axes in ggplot that come from 2 different data frames?

**Comment by oganm** [comment]
Yes but Hadley (creator of ggplot2) wouldn't like you doing it, it is not often recommended in science, and it is not particularly straightforward. Here's a guy describing how to do it https://rpubs.com/kohske/dual_axis_in_ggplot2

**Comment by MayMAAhmed** [comment]
Thanks, I meant more of just specifiying an x axis from a data frame and the y axis from another.

**Comment by oganm** [comment]
But... why? if they don't have the same size, it doesn't make much sense, if they do then just merge the two columns to make a data frame and plot that instead.
```r
newFrame = cbind(frame1$thisIsMyX, frame2$thisIsMyY)
newFrame %>% ggplot ....
```
If you are really against them touching each other you can techically do
```r
ggplot(uselessDataFrame, aes(x = frame1$thisIsMyX, y = frame2$thisIsMyY))
```
There uselessDataFrame will be ignored as you provided exact values to x and y but you are not really supposed to do this
